src/base/utils/misc.cpp

```diff
@@ -642,7 +642,7 @@ QString Utils::Misc::osName()
 {
     // static initialization for usage in signal handler
     static const QString name =
-#ifdef QBT_USES_QT5
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 4, 0))
```

Comment:
@Chocobo1:  Why was this changed?  Was `QSysInfo` (or one of its parameters) introduces in Qt 5.4?